packages/@react-spectrum/table/src/TreeGridTableView.tsx

```diff
@@ -1387,16 +1390,21 @@ function TableCheckboxCell({cell}) {
   );
 }
 
-function TableCell({cell}) {
-  let {state} = useTableContext();
+function TableCell({cell, density, scale}) {
+  let {state, toggleKey, direction} = useTableContext();
   let ref = useRef();
   let columnProps = cell.column.props as SpectrumColumnProps<unknown>;
   let isDisabled = state.disabledKeys.has(cell.parentKey);
   let {gridCellProps} = useTableCell({
     node: cell,
     isVirtualized: true
   }, state, ref);
-
+  let showExpandCollapseButton = cell.index === 0 && (state.collection.getItem(cell.parentKey)?.props.childItems?.length > 0 || state.collection.getItem(cell.parentKey)?.props.children.length > state.collection.columnCount);
```

Comment:
The check on `cell.index === 0` implies that the chevron can only be on the first cell right? I believe there was some discussion about making the chevron attached to the cell which is the first rowColumn header instead, forgot to share that with you, sorry bout that

Reply:
addressed